modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -84,16 +84,29 @@ spec:
       replacement: '$2:$1'
       sourceLabels: ["__meta_kubernetes_pod_annotation_prometheus_io_port","__meta_kubernetes_pod_ip"]
       targetLabel: "__address__"
-    - action: labeldrop
-      regex: "__meta_kubernetes_pod_label_(.+)"
+    - sourceLabels: ["__meta_kubernetes_pod_label_app_kubernetes_io_name","__meta_kubernetes_pod_label_app"] # <2>
+      separator: ";"
+      targetLabel: "app"
+      action: replace
+      regex: "(.+);.*|.*;(.+)"
+      replacement: "${1}${2}"
+    - sourceLabels: ["__meta_kubernetes_pod_label_app_kubernetes_io_version","__meta_kubernetes_pod_label_version"] # <3>
+      separator: ";"
+      targetLabel: "version"
+      action: replace
+      regex: "(.+);.*|.*;(.+)"
+      replacement: "${1}${2}"
     - sourceLabels: ["__meta_kubernetes_namespace"]
       action: replace
       targetLabel: namespace
-    - sourceLabels: ["__meta_kubernetes_pod_name"]
-      action: replace
-      targetLabel: pod_name
+    - action: replace
+      replacement: "the-mesh-identification-string" # <4>
+      targetLabel: mesh_id 
 ----
-<1> Because {ocp-product-title} monitoring ignores the `namespaceSelector` spec in `ServiceMonitor` and `PodMonitor` objects, you must apply the `PodMonitor` object in all mesh namespaces, including the Istio control plane namespace.
+<1> Specifies that the `PodMonitor` object must be applied in all mesh namespaces, including the Istio control plane namespace, because {ocp-product-title} monitoring ignores the `namespaceSelector` spec in `ServiceMonitor` and `PodMonitor` objects.
+<2> Specify the `app` label from `app.kubernetes.io/name` or fallback to `app`.
+<3> Specify the `version` label from `app.kubernetes.io/version` or fallback to `version`.
```

Comment:
Sorry - I'm missing these little things... but now that I am paying very close attention, I noticed this.

Similar to the other change, these `<2>` and `<3>` were just for developer notes. There is nothing for the user to specify here. What these notes could say is just inform the user that this allows for the label values to come from either app.kubernetes.io/[name or version] but if those are missing they will fall back to using [app or version].

But again there is no user-action required here. These were just developer notes/comments. So you could remove `<3>` and `<4>` with no harm done.